models/answers.ts

```diff
@@ -0,0 +1,75 @@
+const admin = require("firebase-admin");
+const firestore = require("../utils/firestore");
+const answerModel = firestore.collection("answers");
+import { Answer, AnswerBody, AnswerFieldsToUpdate } from "../typeDefinitions/answers";
+const { ANSWER_STATUS } = require("../constants/answers");
+
+const createAnswer = async (answerData: AnswerBody) => {
+  try {
+    const { eventId: event_id, answeredBy: answered_by, answer, questionId: question_id } = answerData;
+    const answerRef = answerModel.doc(answerData.id);
+    const createdAndUpdatedAt = admin.firestore.Timestamp.now();
+
+    await answerRef.set({
+      answer,
+      event_id,
+      answered_by,
+      question_id,
+      status: ANSWER_STATUS.PENDING,
+      reviewed_by: null,
+      created_at: createdAndUpdatedAt,
+      updated_at: createdAndUpdatedAt,
+    });
```

Comment:
Could you clarify which naming convention we are using - camelCase or snake_case?

Reply:
`snake_case` for db values
camelCase for all other code, types etc.